posts/2022-07-20-liberty-tools-eclipse.adoc

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "Introducing Liberty Tools for Eclispe"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/awisniew90
+author_github: https://github.com/awisniew90
+seo-title: Introducing Liberty Tools for Eclispe
+seo-description: A next generation plugin that brings the power of Open Liberty into your favorite IDE
+blog_description: "A next generation plugin that brings the power of Open Liberty into your favorite IDE"
+open-graph-image: https://openliberty.io/img/blog/liberty-tools-actions.png
+---
+= Introducing Liberty Tools for Eclipse
+Adam Wisniewski <https://github.com/awisniew90>
+
+Liberty Tools for Eclipse provides a new experience for developing applications with Open Liberty. This early release offers Language Server integration, a Liberty Dashboard for organizing your projects, and "dev mode" from right within the IDE. 
+
+Check out a quick demo:
+
+< Replace link with new demo link >
+++++
+<iframe width="560" height="315" align="center" src="https://www.youtube.com/embed/e_oIInKFtHw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+++++
+
+== Liberty Dashboard
+The Liberty Dashboard keeps all of your Maven and Gradle projects organized and is automatically populated with projects that are configured to run with Open Liberty. With a few clicks, you can start and stop your app, run tests and view test reports. 
+
+== Dev Mode
+
+Liberty Dev mode automatically picks up your code changes and deploys them to your running app all without having to restart the server. With Liberty Tools, your app starts in dev mode automatically giving you all of the benefits of faster development. 
+
+== Language Server Integration
```

Comment:
Same as before: these are two things:  MicroProfile app support, server config support... I don't think LS integration is good.